ios/RNGestureHandlerActionType.h

```diff
@@ -0,0 +1,8 @@
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, RNGestureHandlerActionType) {
```

Comment:
I think it'd be ok to move it to GestureHandler.h

Reply:
We did it analogously to `RNGestureHandlerState.h`.